.gitignore

```diff
@@ -18,6 +18,7 @@ temp-testng-customsuite.xml
 test-output
 .externalToolBuilders
 *~
+\#*
```

Comment:
What is this? What kind of files starts with `#`?

Reply:
Emacs and other editors use them in temp filenames

Reply:
Have you ever tried to use `vim`? It does not cause this issue... :D

Reply:
@williame (I unresolved converstaion) While I'm Emacs user, I don't agree with including this change to this PR since it's unrelated to influxdb. If you still include it, I would recommend separating commits.